be/src/Project/Web/Seller/Seller.Web/Areas/Products/Repositories/CategoriesRepository.cs

```diff
@@ -250,5 +251,30 @@ public async Task SaveAsync(string token, string language, Guid? id, CategorySch
                 throw new CustomException(response.Message, (int)response.StatusCode);
             }
         }
+
+        public async Task<bool> CategoriesSchemasImplementAttibuteAsync(string token, string language, Guid? attributeId)
+        {
+            var apiRequest = new ApiRequest<RequestModelBase>
+            {
+                Language = language,
+                Data = new RequestModelBase(),
+                AccessToken = token,
+                EndpointAddress = $"{_settings.Value.CatalogUrl}{ApiConstants.Catalog.CategorySchemasApiEndpoint}/ImplementAttribute/{attributeId}"
```

Comment:
**suggestion:** Add new variable to ApiConstants with value: /api/v1/CategorySchemas/ImplementAttribute and meaningful name